tests/config_test.py

```diff
@@ -85,3 +85,18 @@ def test_configure_from_object_changes_scribe_disable(self):
         )
         out = proc.communicate()[0].decode('UTF-8')
         assert out == 'True\nFalse\n'
+
+    def test_logging_not_configured(self):
+        proc = subprocess.Popen(
+            (
+                sys.executable, '-c',
+                'import clog\n'
+                'try:\n'
+                '   clog.log_line("foo", "bar")\n'
+                'except Exception as e:\n'
+                '   print(e.__class__.__name__)\n'
+            ),
+            stdout=subprocess.PIPE,
+        )
```

Comment:
I'm not sure why the travis build is failing. It says the tox command can't be found. Any ideas?